aip/general/0162.md

```diff
@@ -195,31 +176,15 @@ message ListBookRevisionsRequest {
 
 message ListBookRevisionsResponse {
   // The revisions of the book.
-  repeated Book books = 1;
+  repeated BookRevision books = 1;
 
   // A token that can be sent as `page_token` to retrieve the next page.
   // If this field is omitted, there are no subsequent pages.
   string next_page_token = 2;
 }
 ```
 
-While revision listing methods are mostly similar to standard `List` methods
-([AIP-132][]), the following important differences apply:
-
-- The first field in the request message **must** be called `name` rather than
-  `parent` (this is listing revisions for a specific book, not a collection of
-  books).
-- The URI **must** end with `:listRevisions`.
-- Revisions **must** be ordered in reverse chronological order. An `order_by`
-  field **should not** be provided.
-- The returned resources **must** include an explicit revision ID in the
-  resource's `name` field.
-  - If providing the full resource is expensive or infeasible, the revision
-    object **may** only populate the `string name`, `string revision_id`, and
-    `google.protobuf.Timestamp revision_create_time` fields instead. The
-    `string name` field **must** include the resource name and an explicit
-    revision ID, which can be used for an explicit `Get` request. The API
-    **must** document that it will do this.
+- Revisions must be ordered in reverse chronological order. An `order_by` field **should not** be provided in the list request.
```

Comment:
is this necessary? I'm not clear on why one couldn't order revisions inf they wanted to.

Reply:
I think this is a useful default behavior. Updated the sentence. WDYT?

Reply:
We definitely should have an `order_by` field - not having it produces exceptions in things like the API linter and also force tooling to handle partial List endpoints.

I'm fine with keeping the guidance on reverse chronological order for now (and btw makes sure to use `**must**` to emphasize it's and RFC-2119 term.